definitions/marts/npq_separation_prep/npq_enrolments_v2.sqlx

```diff
@@ -83,13 +83,25 @@ config {
     }
 }
 
+-- Gather the first user profile per user_id to obtain the TRN associated with it. This value will be used as the application_trn before any verification checks have taken place.
+WITH first_trns AS (

```

Comment:
I have concerns about linking this to the user only and not finding a way to connect it with the application. What would happen if this individual was to re-apply and mis-enter their TRN? Is there any way to capture that, could that happen?

Reply:
I have reworked this section on npq_enrolments. Please review and let me know your thoughts on the proposed method.